testplan/testing/multitest/result.py

```diff
@@ -101,13 +101,25 @@ def __exit__(self, exc_type, exc_value, tb):
 assertion_state = threading.local()
 
 
-def report_target(func: Callable) -> Callable:
+def report_target(func: Callable, ref_func: Callable = None) -> Callable:
     """
     Sets the decorated function's filepath and line-range in assertion state.
+    If the target function is a parametrized function, should refer to its
+    parametrized template to find information of the original function.
```

Comment:
Can you please add this to a `:param ref_func:` directive? Given this is a decorator, I doubt the `func` parameter or the return value needs to be explained at docstring level, but I think the `ref_func` is an important and non-trivial addition to the decorator. Also helps in IDE.

Reply:
doc string added.